.github/workflows/nodejs.yml

```
@@ -1,28 +1,104 @@
-name: Node CI
+name: CI/CD Pipeline
 
-on: [push]
+on: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)